livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/tts.py

```diff
@@ -32,7 +33,7 @@
 from .models import TTSModels, TTSVoices
 from .utils import AsyncAzureADTokenProvider
 
-OPENAI_TTS_SAMPLE_RATE = 24000
+OPENAI_TTS_SAMPLE_RATE = 48000
```

Comment:
48000? Or it's just the opus default?

Reply:
this is the opus default, and also what OAI is returning